apps/api/v2/src/modules/organizations/repositories/attributes/organization-attribute-option.repository.test.ts

```diff
@@ -0,0 +1,180 @@
+import { AssignOrganizationAttributeOptionToUserInput } from "@/modules/organizations/inputs/attributes/assign/organizations-attributes-options-assign.input";
+import { CreateOrganizationAttributeOptionInput } from "@/modules/organizations/inputs/attributes/options/create-organization-attribute-option.input";
+import { UpdateOrganizationAttributeOptionInput } from "@/modules/organizations/inputs/attributes/options/update-organizaiton-attribute-option.input.ts";
+import { OrganizationAttributeOptionRepository } from "@/modules/organizations/repositories/attributes/organization-attribute-option.repository";
+import { OrganizationAttributesService } from "@/modules/organizations/services/attributes/organization-attributes.service";
+import { PrismaReadService } from "@/modules/prisma/prisma-read.service";
+import { PrismaWriteService } from "@/modules/prisma/prisma-write.service";
+import { Test, TestingModule } from "@nestjs/testing";
+
+describe("OrganizationAttributeOptionRepository", () => {
```

Comment:
I think e2e tests for controllers would be more valuable, we are mostly testing prisma here